string-manipulation-3/src/reverse-words.js

```diff
@@ -0,0 +1,20 @@
+/* exported reverseWords */
+
+function reverseWords(string) {
+  var strArr = string.split(' ');
+  var tempReverseArr = [];
+  for (var k = 0; k < strArr.length; k++) {
+    var temp = [];
+    for (var i = 0; i < strArr[k].length; i++) {
+      temp.unshift(strArr[k][i]);
+    }
+    tempReverseArr.push(temp);
+  }
+  var reverseArr = [];
+  for (var m = 0; m < tempReverseArr.length; m++) {
+    var newStr = tempReverseArr[m].join('');
+    reverseArr.push(newStr);
+    var reverse = reverseArr.join(' ');
+  }
```

Comment:
You can probably do this without the 3rd for loop. HINT: try looping through the string backwards to reverse the string so you don't need the extra array